typescript/big-practice/src/scripts/models/userModel.ts

```diff
@@ -0,0 +1,111 @@
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Dont need to use `uuid` lib anymore, you could use encrypt API `crypto.randomUUID()`